### ISSUE TITLE
node registration fails after uninstall/install steps

### ISSUE THREAD

**Issue by harshavardhana** [minio/direct-csi#167]
```
Containers:
  node-driver-registrar:
    Container ID:  docker://898bef3eb188b934e746ec1f767e2777986f16989b18ff2c46dfcdc27071e149
    Image:         quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
    Image ID:      docker-pullable://quay.io/k8scsi/csi-node-driver-registrar@sha256:9622c6a6dac7499a055a382930f4de82905a3c5735c0753f7094115c9c871309
    Port:          <none>
    Host Port:     <none>
    Args:
      --v=3
      --csi-address=unix:///csi/csi.sock
      --kubelet-registration-path=/var/lib/kubelet/plugins/direct-csi-min-io/csi.sock
    State:      Terminated
      Reason:   Error
      Message:  I0207 07:12:22.904132   79009 main.go:110] Version: v1.3.0-0-g6e9fff3e
I0207 07:12:22.904209   79009 main.go:120] Attempting to open a gRPC connection with: "unix:///csi/csi.sock"
I0207 07:12:22.904233   79009 connection.go:151] Connecting to unix:///csi/csi.sock
I0207 07:12:22.904792   79009 main.go:127] Calling CSI driver to discover driver name
I0207 07:12:22.905381   79009 main.go:137] CSI driver name: "direct-csi-min-io"
I0207 07:12:22.905446   79009 node_register.go:51] Starting Registration Server at: /registration/direct-csi-min-io-reg.sock
I0207 07:12:22.905573   79009 node_register.go:60] Registration Server started at: /registration/direct-csi-min-io-reg.sock
I0207 07:12:24.547144   79009 main.go:77] Received GetInfo call: &InfoRequest{}
I0207 07:12:24.896058   79009 main.go:87] Received NotifyRegistrationStatus call: &RegistrationStatus{PluginRegistered:false,Error:RegisterPlugin error -- plugin registration failed with err: error updating Node object with CSI driver node info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.min.io/identity":"direct-csi-min-io" but existing label is "direct.csi.min.io/identity":"direct.csi.min.io",}
E0207 07:12:24.896143   79009 main.go:89] Registration process failed with error: RegisterPlugin error -- plugin registration failed with err: error updating Node object with CSI driver node info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.min.io/identity":"direct-csi-min-io" but existing label is "direct.csi.min.io/identity":"direct.csi.min.io", restarting registration container.

      Exit Code:  1
      Started:    Sat, 06 Feb 2021 23:12:22 -0800
      Finished:   Sat, 06 Feb 2021 23:12:24 -0800
    Last State:   Terminated
      Reason:     Error
      Message:    I0207 07:11:39.915362   78347 main.go:110] Version: v1.3.0-0-g6e9fff3e
I0207 07:11:39.915449   78347 main.go:120] Attempting to open a gRPC connection with: "unix:///csi/csi.sock"
I0207 07:11:39.915472   78347 connection.go:151] Connecting to unix:///csi/csi.sock
I0207 07:11:39.915955   78347 main.go:127] Calling CSI driver to discover driver name
I0207 07:11:39.916421   78347 main.go:137] CSI driver name: "direct-csi-min-io"
I0207 07:11:39.916489   78347 node_register.go:51] Starting Registration Server at: /registration/direct-csi-min-io-reg.sock
I0207 07:11:39.916622   78347 node_register.go:60] Registration Server started at: /registration/direct-csi-min-io-reg.sock
I0207 07:11:41.535469   78347 main.go:77] Received GetInfo call: &InfoRequest{}
I0207 07:11:41.888733   78347 main.go:87] Received NotifyRegistrationStatus call: &RegistrationStatus{PluginRegistered:false,Error:RegisterPlugin error -- plugin registration failed with err
: error updating Node object with CSI driver node info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.mi
n.io/identity":"direct-csi-min-io" but existing label is "direct.csi.min.io/identity":"direct.csi.min.io",}
E0207 07:11:41.888832   78347 main.go:89] Registration process failed with error: RegisterPlugin error -- plugin registration failed with err: error updating Node object with CSI driver node
 info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.min.io/identity":"direct-csi-min-io" but existing l
abel is "direct.csi.min.io/identity":"direct.csi.min.io", restarting registration container.
```

**Comment by immanuelfodor** [comment]
I get the same on a fresh install:

```bash
$ k logs direct-csi-min-io-8gvwh node-driver-registrar 
I0319 17:48:04.525287   74736 main.go:113] Version: v2.1.0-0-g80d42f24
I0319 17:48:04.525725   74736 main.go:137] Attempting to open a gRPC connection with: "unix:///csi/csi.sock"
I0319 17:48:04.525744   74736 connection.go:153] Connecting to unix:///csi/csi.sock
I0319 17:48:04.527501   74736 main.go:144] Calling CSI driver to discover driver name
I0319 17:48:04.528379   74736 main.go:154] CSI driver name: "direct-csi-min-io"
I0319 17:48:04.528453   74736 node_register.go:52] Starting Registration Server at: /registration/direct-csi-min-io-reg.sock
I0319 17:48:04.528578   74736 node_register.go:61] Registration Server started at: /registration/direct-csi-min-io-reg.sock
I0319 17:48:04.528683   74736 node_register.go:83] Skipping healthz server because HTTP endpoint is set to: ""
I0319 17:48:06.112082   74736 main.go:80] Received GetInfo call: &InfoRequest{}
I0319 17:48:06.449576   74736 main.go:90] Received NotifyRegistrationStatus call: &RegistrationStatus{PluginRegistered:false,Error:RegisterPlugin error -- plugin registration failed with err: error updating Node object with CSI driver node info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.min.io/identity":"direct-csi-min-io" but existing label is "direct.csi.min.io/identity":"direct.csi.min.io",}
E0319 17:48:06.449608   74736 main.go:92] Registration process failed with error: RegisterPlugin error -- plugin registration failed with err: error updating Node object with CSI driver node info: error updating node: timed out waiting for the condition; caused by: detected topology value collision: driver reported "direct.csi.min.io/identity":"direct-csi-min-io" but existing label is "direct.csi.min.io/identity":"direct.csi.min.io", restarting registration container.

$ k get po
NAME                                 READY   STATUS             RESTARTS   AGE
direct-csi-min-io-7695b55b7f-4zwhs   2/2     Running            0          30m
direct-csi-min-io-7695b55b7f-jvfx6   2/2     Running            0          30m
direct-csi-min-io-7695b55b7f-t2trt   2/2     Running            0          30m
direct-csi-min-io-8gvwh              2/3     CrashLoopBackOff   10         30m
direct-csi-min-io-slmxn              2/3     CrashLoopBackOff   10         30m
direct-csi-min-io-wt49s              2/3     CrashLoopBackOff   10         30m

$ k krew info direct-csi
NAME: direct-csi
INDEX: default
URI: https://github.com/minio/direct-csi/releases/download/v1.2.8/kubectl-direct_csi_linux_amd64.zip
SHA256: 40177b04937740e2cdc443bdba982d049096aff42ca43123f1455721e6a982bd
VERSION: v1.2.8
HOMEPAGE: https://github.com/minio/direct-csi
DESCRIPTION: 
This plugin deploys and manages the lifecycle of DirectCSI driver - a driver
that creates volumes from drives available in the kubernetes cluster
```

**Comment by immanuelfodor** [comment]
I think I had it once installed via Helm but then removed it, and now when I tried the install with the krew plugin, it seems it conflicted with the old labels/annotations. Removing these made all pods running upon a new install, and the new labels are created on the nodes fine. Although the identity label could be enough, I removed all to let it recreate them in case there are some mismatches. The only observable difference after the install is with the new `direct.csi.min.io/identity: direct-csi-min-io` label.

```bash
k label nodes --all direct.csi.min.io/identity- direct.csi.min.io/node- direct.csi.min.io/rack- direct.csi.min.io/region- direct.csi.min.io/zone-
k annotate nodes --all csi.volume.kubernetes.io/nodeid-
```
